docs/language/operators/join.md

```diff
@@ -6,48 +6,91 @@
 
 ```
 <left-input>
-| [anti|inner|left|right] join (
+| [anti|inner|left|right] join [as { <left-name>,<right-name> }] (
```

Comment:
should there be the ability to do an anti right join?

Reply:
Yeah I don't think I have this quite right